batch7crud-roll1/src/main/java/com/lftechnology/batch7crud/validator/Validator.java

```diff
@@ -0,0 +1,16 @@
+package com.lftechnology.batch7crud.validator;
+
+import com.lftechnology.batch7crud.exception.ValidationException;
+
+import java.util.Map;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.Map'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)